www/scipylib/building/windows.rst

```diff
@@ -32,185 +32,231 @@ information than this document will provide.
+set to :code:`C:\\opt`, then you might see a line like this in the MSYS2 terminal:
+
+.. code:: shell
+
+   Copying the static library to /c/opt/64/lib
+
+Installing OpenBLAS
+===================
+
+If you see that line, then you might have OpenBLAS correctly, even if other failures
+occurred. Look in that folder for :code:`openblas.a`. If you find a file called something
+like :code:`libopenblas_5f998ef_gcc7_2_0.a`, just rename it to `openblas.a` and continue.
+If the file isn't there, then poke around and try to find the file elsewhere. If you don't
+have that file, you'll probably need to find out what happened and then build OpenBLAS
+again. But if you have that file, we'll assume that you've completed this step correctly.
+Proceeding on that assumption, let's build SciPy.
+
+**Before continuing, it is extremely important that you remove every file named either**
+:code:`openblas.lib` **or** :code:`libopenblas.lib` **from your computer. Failure to do**
+**so could result in later build errors that will be difficult to debug. If you see the**
+**following output at any point, you have not completed this step correctly.**
+
+.. code:: shell
+
+   FOUND:
+      libraries = ['libopenblas', 'libopenblas']
+      library_dirs = ['C:\opt\64\lib']
+      language = c
+      define_macros = [('HAVE_CBLAS', None)]
+   FOUND:
+      libraries = ['libopenblas', 'libopenblas']
+      library_dirs = ['C:\opt\64\lib']
+      language = c
+      define_macros = [('HAVE_CBLAS', None)]
+
+Building SciPy
+==============
+
+From this point forward, we're not going to need MSYS2 any longer, so you might uninstall
+it to prevent further confusion. Building SciPy requires a different set of build tools
+than building OpenBLAS (yes, the whole previous excercise was to build a single file:
+:code:`openblas.a`), so let's go ahead and install them:
+
+1) Install MinGW-w64 from https://mingw-w64.org. Use the "MinGW builds" and 
+   check the "posix-threads" box.
+2) Install Microsoft Visual Studio 2015 or 2017 Community Edition (use the `build tools`_
+   from Microsoft)
+3) Install git from https://git-scm.org/
+4) Finally, install Python from https://python.org/ (make sure to check the box to install
+   pip)
+
+After you've installed the required software, open Powershell (click start, type "powershell",
+press enter), change to a good location to build (just like with building OpenBLAS, but this
+time we're using a different toolchain with different commands), and clone SciPy. From now
+on, we'll use powershell for the rest of the procedure.
+
+.. code:: shell
+
+   cd C:\Users\MyUser\Downloads
+   git clone https://github.com/scipy/scipy.git
+   cd scipy
+   
+Now we need to copy the :code:`openblas.a` file that we've built earlier to the correct
+location. Find where Python is installed:
+
+.. code:: shell
+
+   python -c "import sys; print(sys.executable)"
+
+If your Python is installed somewhere like :code:`C:\\Program Files\\Python36\\python.exe`,
```

Comment:
Typical place for Win10 and Py3.6 is 

    C:\Users\<user name>\AppData\Local\Programs\Python\Python36\Lib